tests/yam/validate/post_registration.pm

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Check registration status and reregister system via suseconnect tool.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    assert_script_run "SUSEConnect -s | grep 'Not Registered'";
+    assert_script_run "SUSEConnect --status-text";
+
```

Comment:
no need for this blank line either. or add back the line above 21.

Reply:
I removed the blank line, thanks.